src/components/ProgressForm/ProgressHeader.tsx

```diff
@@ -17,8 +27,27 @@ const ProgressHeader: FC<progressHeaderProps> = ({
                     {updateType} updates
                 </p>
                 <p className={styles.bannerPara}>
-                    Let&apos;s try to avoid having zero days
+                    Let us try to avoid having zero days
                 </p>
+                {updateType === STANDUP ? (
```

Comment:
`updateType === STANDUP && ()` is enough.

Ternary operator is unnecessary here.

Reply:
https://github.com/Real-Dev-Squad/website-status/pull/710#discussion_r1268907385
I tried but was getting error and did as suggested by @vvaibhavdesai